scripts/templates.ts

```diff
@@ -24,12 +24,22 @@ function _runTemplate(inputPath: string, outputPath: string, logger: logging.Log
     encode: (x: string) => global.encodeURIComponent(x),
     require: (x: string) => require(path.resolve(path.dirname(inputPath), x)),
   });
+
+  const fileExists = fs.existsSync(outputPath);
```

Comment:
This is done in CI using `git` to do the check (check `validate-templates`). The user should not have to delete files before overwriting it.

This change is really inconvenient for the user. This script is not a validation, it's just generating the templates.

Reply:
Actually the validation is in `scripts/validate.ts`.